src/features/community/communitySlice.ts

```diff
@@ -97,11 +97,15 @@ export const getFavoriteCommunities =
 
     if (!userHandle) return;
 
-    const communities = await db.getSetting("favorite_communities", {
-      user_handle: userHandle,
-    });
-
-    dispatch(setFavorites(communities));
+    try {
+      const communities = await db.getSetting("favorite_communities", {
+        user_handle: userHandle,
+      });
+
+      dispatch(setFavorites(communities));
+    } catch (e) {
+      // swallow the error, probably "Setting not found"
```

Comment:
btw this PR looks awesome! :) Thank you!

For this error - is it an instance of something? Can we do `if (!(e instanceof SomeError)) throw e;` so we don't hide genuine errors?

Reply:
Or perhaps a guard would look nicer

```js
} catch (error) {
  if (error instanceof SomeError) return;

  throw error;
}
```

Reply:
yeah that makes sense but currently it's a generic promise rejection error. I think that could be the topic of another pr :)

I just added this here because it was logging unhandled errors to the console.